### PR TITLE
enable kyverno-connectivity policy

### PR DESCRIPTION
**Pull request by bavarianbidi**
towards https://github.com/giantswarm/roadmap/issues/1419

### CODE REVIEW

File: flux-manifests/kyverno-policies-connectivity.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+api_version: generators.giantswarm.io/v1
+app_catalog: control-plane-catalog
+app_destination_namespace: giantswarm
+app_name: kyverno-policies-observability
```
Comment (glitchcrab) [inline flux-manifests/kyverno-policies-connectivity.yaml:4]: should all instances of `observability` not be changed to `connectivity`?

Comment (bavarianbidi) [reply flux-manifests/kyverno-policies-connectivity.yaml]: :see_no_evil: expert tip: safe the file after changing the content :see_no_evil:

Comment (glitchcrab) [reply flux-manifests/kyverno-policies-connectivity.yaml]: :sweat_smile:

**Review by glitchcrab — APPROVED** [review]
seal_of_approval

**Pull request merged**